### PR TITLE
Minor fixes and improvements to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by koraktor**
Fixing a few typos, improving the formatting and other improvements.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,9 +36,9 @@ can be applied as quickly as possible:
    modify the `README.md` file to reflect that. Again, if you don't
    update the `README`, we have to, and this holds up acceptance.
 
-4. **Update the Changelog (`CHANGES`):** The changelog helps give an
+4. **Update the changelog (`CHANGES`):** The changelog helps give an
```
Comment (kytrinyx) [inline CONTRIBUTING.md]: This is the only change that caught my eye.

I'm still trying to figure out when it's a Changelog, when to call it a change log, and when to refer to the actual CHANGELOG.

http://keepachangelog.com/

Thoughts?

**Comment by kytrinyx** [comment]
Good catch on s/Brazillian/Brazilian/!

This all looks good to me, with one question, asked inline.

File: CONTRIBUTING.md (hunk 1 above)
Comment (koraktor) [inline CONTRIBUTING.md]: Thanks for the hint. I added another commit.

**Comment by kytrinyx** [comment]
@TrevorBramble @zzak @ashleygwilliams I'm happy with the changelog -> change log edit. If you hate it, would you speak up, if you like it would you merge?

**Comment by ashleygwilliams** [comment]
:+1: from me. happy to merge, but i'll give everyone a sec to weigh in

**Comment by TrevorBramble** [comment]
These are great changes. :+1: 

If anything I'd like to see the `CHANGES` file renamed to `CHANGELOG`, but that's not relevant to this pull request. =^)

**Comment by ashleygwilliams** [comment]
totes agree @TrevorBramble ... maybe file and issue and we can tweet it out as a good "your-first-pr" ? (actually, i can just do that.)

**Comment by ashleygwilliams** [comment]
merging.

**Comment by zzak** [comment]
LGTM

**Pull request merged**